prow/pvsc/GoogleCloudPlatform_esp-v2_pvsc.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
```

Comment:
a) PVCs (c and s are transposed)
b) prow/build/pvcs (this is for the build cluster)
c) Update the rule to apply this: https://github.com/GoogleCloudPlatform/oss-test-infra/blob/29de34218ad9075ee82feabc558b03b20bb26960/prow/Makefile#L37-L40